webapp/channels/src/plugins/test/post_type.test.tsx

```diff
@@ -8,11 +8,9 @@ import {Preferences} from 'mattermost-redux/constants';
 
 import PostMessageView from 'components/post_view/post_message_view/post_message_view';
 
-class PostTypePlugin extends React.PureComponent {
-    render() {
-        return <span id='pluginId'>{'PostTypePlugin'}</span>;
-    }
-}
+const PostTypePlugin: React.FunctionComponent = () => (
```

Comment:
```suggestion
const PostTypePlugin = () => (
```
nit we can remove that